scripts/pylib/twister/twisterlib/handlers.py

```diff
@@ -512,6 +512,9 @@ def _create_command(self, runner, hardware):
                     elif runner == "openocd" and product == "EDBG CMSIS-DAP":
                         command_extra_args.append("--cmd-pre-init")
                         command_extra_args.append("cmsis_dap_serial %s" % board_id)
+                    elif runner == "openocd" and product == "LPC-LINK2 CMSIS-DAP":
+                        command_extra_args.append("--cmd-pre-init")
+                        command_extra_args.append("adapter serial %s" % board_id)
```

Comment:
Could you please also apply this changes in:
https://github.com/zephyrproject-rtos/zephyr/blob/1cfe90d825a1e07d9aa77bf389338dc72dfda6fe/scripts/pylib/pytest-twister-harness/src/twister_harness/device/hardware_adapter.py#L64
there is same method but used by by pytest.

Reply:
thanks to reminder this change, I add